decred/tests/unit/dcr/test_txscript.py

```diff
@@ -2352,30 +2352,84 @@ def test_scriptNumBytes(self):
-        self.assertEqual(pi.poolFees, rePI.poolFees)
-        self.assertEqual(pi.script, rePI.script)
-        self.assertEqual(pi.ticketAddress, rePI.ticketAddress)
-        self.assertEqual(pi.voteBits, rePI.voteBits)
-        self.assertEqual(pi.voteBitsVersion, rePI.voteBitsVersion)
-        self.assertEqual(pi.unixTimestamp, rePI.unixTimestamp)
+def test_is_dust_amount():
+    class test:
+        def __init__(
+            self, name, value, relayFee, isDust,
+        ):
+            """
+            Args:
+                name (str): Short description of the test.
+                value (int): tx output value.
+                relayFee (int): Network relay fee.
+                isDust (bool): whether this compinatin of size, value, and
+                    relayFee is considered dust.
+            """
+            self.name = name
+            self.value = value
+            self.relayFee = relayFee
+            self.isDust = isDust
+
+    tests = [
+            #test(
+            #    # Any value is allowed with a zero relay fee.
+            #    "zero value with zero relay fee",
+            #    0,
+            #    0,
+            #    True,
+            #),
+            test(
+                # Zero value is dust with any relay fee"
+                "zero value with very small tx fee",
+                0,
+                1,
+                True,
+            ),
+            test(
+                "25 byte public key script with value 602, relay fee 1e3",
+                602,
+                1000,
+                True,
+            ),
+            test(
+                "25 byte public key script with value 603, relay fee 1e3",
+                603,
+                1000,
+                False,
+            ),
+            test(
+                "25 byte public key script with value 60299, relay fee 1e5",
+                60299,
+                1e5,
+                True,
+            ),
+            test(
+                "25 byte public key script with value 60300, relay fee 1e5",
+                60300,
+                1e5,
+                False,
+            ),
+            test(
+                "25 byte public key script with value 6029, relay fee 1e4",
+                6029,
+                1e4,
+                True,
+            ),
+            test(
+                "25 byte public key script with value 6030, relay fee 1e4",
+                6030,
+                1e4,
+                False,
+            ),
+            test(
+                # Maximum allowed value is never dust.
+                "max amount is never dust",
+                txscript.MaxAmount,
+                txscript.MaxAmount,
+                False,
+            ),
+        ]
+    size = 25
+    for test in tests:
+        res = txscript.isDustAmount(test.value, size, test.relayFee)
+        assert res == test.isDust, "expecte \"{}\" for test \"{}\" but got \"{}\"".format(test.isDust, test.name, res)
```

Comment:
s/expecte/Expected/
You can avoid the backslashes by using both single and double quote chars, and avoid the `format` call too:
`f'Got {res} instead of {test.isDust} in test "{test.name}"'`